src/System.CommandLine.DragonFruit.Tests/ConfigureFromMethodTests.cs

```diff
@@ -235,6 +235,17 @@ public async Task Method_parameters_on_the_invoked_member_method_are_bound_to_ma
             _receivedValues.Should().BeEquivalentTo(true);
         }
 
+        [Fact]
+        public async Task Method_with_multiple_parameters_with_default_values_are_resolved_correctly()
```

Comment:
It looks like the bug is in `System.CommandLine` but the test is in `System.CommandLine.DragonFruit`. I'm assuming that means the same bug can express itself in non-DragonFruit code as well, so maybe add a test for this in `System.CommandLine.Tests`?

Reply:
Done